charts/renku/requirements.yaml

```diff
@@ -19,7 +19,7 @@ dependencies:
   version: 0.14.4
   repository: "@stable"
 - name: keycloak
-  version: 3.2.0
+  version: 4.0.1
```

Comment:
Stage it for test

Reply:
done; we've arrived here now, it seems that v4 has a few changes to consider here:
https://stackoverflow.com/questions/45352880/keycloak-invalid-parameter-redirect-uri

Reply:
Try with 3.4.2 and we can tag this PR with renku 0.3.0. This one shouldn't break things.